app/views/finance/participants/_ecf_profile.html.erb

```diff
@@ -72,48 +68,65 @@
 
 <h4>Induction records</h4>
 
-<% pp.induction_records.order(:start_date).each do |ir| %>
+<% pp.induction_records.order(start_date: :desc).each do |ir| %>
   <h5>Induction record: <%= ir.id %></h5>
 
   <%= govuk_summary_list do |summary_list| %>
     <% summary_list.row do |row| %>
       <% row.key { "Training programme" } %>
       <% row.value { ir.induction_programme.training_programme.humanize } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "School URN" } %>
       <% row.value { ir.induction_programme.school_cohort.school.urn } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "Lead provider" } %>
       <% row.value { ir.induction_programme.partnership&.lead_provider&.name } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "Induction status" } %>
       <% row.value { ir.induction_status } %>
+      <% row.action(text: :none) %>
     <% end %>
 
     <% summary_list.row do |row| %>
       <% row.key { "Training status" } %>
       <% row.value { ir.training_status } %>
+      <%
+        if ir == pp.latest_induction_record_for(cpd_lead_provider: ir.cpd_lead_provider)
+          row.action(
+            text: "Change",
+            visually_hidden_text: "training status",
+            href: new_finance_participant_profile_ecf_induction_records_path(pp.id, ir.id))
```

Comment:
instead of passing in the induction record ID, can we pass in the CPD lead provider ID, which then works well with our services as they already accept a CPD lead provider?

Reply:
Hi Abeer you ok?
Just realised I'm sending the induction record ID up to `app/forms/finance/ecf/change_training_status_form.rb` only to be able to then fetch the correct `cpd_lead_provider` from it and then passing only the `cpd_lead_provider` to the services (Defer, Withdraw , etc..). So it's up for the services to select again the latest induction record for that lead provider.
I hope it's all good
Tks